bootstrap/src/lib/util.ts

```diff
@@ -22,6 +22,15 @@ export const toObjectWithNumbers = (obj: any) => {
   return Object.fromEntries(Object.entries(obj).map(([k, v]) => [k, toNumber(v)]))
 }
 
+// pick a random string after splitting with the delimiter ("a&b&c" "&" -> choice(["a","b","c"]))
+export const pickRandomFromString = (str: string, delimiter: string) => {
+  if (typeof str !== 'string') {
```

Comment:
I think I would be a good opportunity to use the `util.getRequiredEnv` helper. Will help with the undefined Env var too. Something like:
```js
export const getRandomEnv = (name: string, delimiter = ',', prefix = '') => {
    const val = getRequiredEnv(name, prefix)
    const items = str.split(delimiter)
    return items[Math.floor(Math.random() * items.length)]
}
```
And use it like: `const apiKey = util.getRandomEnv('API_KEY')`

Reply:
Ah yep great call, included your suggestion. Don't know how I missed that function while I was working in `util.ts`